EURO2021 abstract

MichalisPanayides opened the pull request.

MichalisPanayides committed:
Write initial form of abstact

drvinceknight commented on EURO2021/Abstract.tex:
```diff
@@ -0,0 +1,45 @@
+\documentclass{article}
+
+\title{A game theoretic model of the behavioural gaming that takes place at the 
+EMS - ED interface}
+\author{Michalis Panayides}
```
```suggestion
\title{A 3 player game theoretic model of a choice between two queueing systems with strategic managerial decision making}
\author{Michalis Panayides}
```

The healthcare part is not the "main" attraction for the EURO talk.

drvinceknight commented on EURO2021/Abstract.tex:
```diff
@@ -0,0 +1,45 @@
+\maketitle
+\begin{abstract}
+    
+    Operational Research (OR) techniques provide a toolkit of mathematical 
+    modelling approaches that are routinely used for problem solving in many
+    sectors. Healthcare is considered to be one of such sectors where, in recent
+    years, OR has found several applications.
```
```suggestion
```

Not necessary for the audience.

MichalisPanayides committed:
Rewrite abstract

Rewrite abstract so that it focuses more on the mathematical
models and formulations rather than the applications.

MichalisPanayides committed:
Minor changes

drvinceknight commented on EURO2021/Abstract.tex:
```diff
@@ -0,0 +1,44 @@
+\documentclass{article}
+
+\title{A 3-player game theoretic model of a choice between two queueing systems with strategic managerial decision making}
```
Nice.

drvinceknight commented on EURO2021/Abstract.tex:
```diff
@@ -0,0 +1,44 @@
+    players.
+
+    The first aspect of this work is the development of a queueing system
+    with two waiting spaces and two types of individuals. 
```
```suggestion
    with two consecutive waiting spaces. The strategic managerial behaviour corresponds to how individuals use these waiting spaces.
```

drvinceknight commented on EURO2021/Abstract.tex:
```diff
@@ -0,0 +1,44 @@
+    The state probabilities of the Markov chain system have been used to extract
+    the performance measures of the queueing model (e.g. mean time in each 
+    waiting room, mean number of individuals in each room, etc.).
+    Given two such systems, the scenario explored is
+    how to distribute individuals among the two in order to optimise certain
+    performance measures. 
```
```suggestion

```

drvinceknight commented on EURO2021/Abstract.tex:
```diff
@@ -0,0 +1,44 @@
+    simulation and Markov chains. 
+    The state probabilities of the Markov chain system have been used to extract
+    the performance measures of the queueing model (e.g. mean time in each 
+    waiting room, mean number of individuals in each room, etc.).
```
```suggestion
    waiting room, mean number of individuals in each room, etc.).
    
```

drvinceknight commented on EURO2021/Abstract.tex:
```diff
@@ -0,0 +1,44 @@
+    normal-form game between the two queueing systems.
+
+    This particular system can be applied in a healthcare scenario where it 
+    could capture the emergent behaviour between the Emergency Medical Service 
```
```suggestion
    captures the emergent behaviour between the Emergency Medical Service 
```

drvinceknight commented on EURO2021/Abstract.tex:
```diff
@@ -0,0 +1,44 @@
+    This particular system can be applied in a healthcare scenario where it 
+    could capture the emergent behaviour between the Emergency Medical Service 
+    (EMS) and the Emergency Department (ED). 
+    The model will be used to form a 3-player game between the EDs of two 
+    neighbouring hospitals and the EMS.
```
```suggestion
    This will be used to investigate the impact of target measures on patient welfare.
```

drvinceknight reviewed (changes requested):
Some minor suggestions.

MichalisPanayides committed:
Apply suggestions from code review

Co-authored-by: Vince Knight <vincent.knight@gmail.com>

MichalisPanayides committed:
Render PDF of the new changes

MichalisPanayides committed:
Convert to markdown

MichalisPanayides committed:
Fix some line breaks

MichalisPanayides committed:
Apply second suggestions from review

MichalisPanayides committed:
Add first slide

Include solarized theme file as well

MichalisPanayides committed:
Add initial story board

MichalisPanayides committed:
Add about me section

MichalisPanayides committed:
Include about me main.tex

MichalisPanayides committed:
Add queues examples slides

MichalisPanayides committed:
Add Discrete Event Simulation Slides

MichalisPanayides committed:
Add Markov chain frames

MichalisPanayides committed:
Add transition matrix frames

Include python script that generates heatmap of comparisons
between simulation and markov steady state probabilities

MichalisPanayides committed:
Change some frame titles

MichalisPanayides committed:
Add slides for performance measures

MichalisPanayides committed:
Add frames for game theoretic model

MichalisPanayides committed:
Add learning algorithms frames

MichalisPanayides committed:
Add frames for code

MichalisPanayides committed:
Remove M|M|c frames

MichalisPanayides committed:
Add more mathematics to performance measure frame

Include additional comparison plots and move them all in smae slide

MichalisPanayides committed:
Add tree diagram representation of the game

MichalisPanayides committed:
Edit learning algorithms frames

MichalisPanayides committed:
Add final slide - "picture slide"

MichalisPanayides committed:
Add lit review in queueing slides

MichalisPanayides committed:
Ethnography meeting - 1 (#7)

* Use slides from EURO talk

* Add exmaple of network

* Add final ethnography slide

* Fix title page

* Remove references from queueing examples

* Remove game - formulation slides

* Add labels to queueing network structure slide

* Reword a sentence

MichalisPanayides committed:
Minor changes and tweaks

- Get rid of parking circle in tikz picture
- Swap L_1 with L_2 in performance measures
- Reword a sentence

MichalisPanayides committed:
Change colormap for heatmaps

MichalisPanayides committed:
Add literature on slide

MichalisPanayides committed:
Swap Proportion within time expressions

MichalisPanayides committed:
Add changes discussed on practice talk

MichalisPanayides committed:
Change notation of proportion of inds within time

MichalisPanayides committed:
Remove unnecessary pause

MichalisPanayides committed:
Hopefully, final minor changes before talk
